py/desispec/workflow/procfuncs.py

```diff
@@ -991,6 +1012,136 @@ def joint_fit(ptable, prows, internal_id, queue, reservation, descriptor, z_subm
 
     return ptable, joint_prow, internal_id
 
+#########################################
+########     Redshifts     ##############
+#########################################
+def submit_redshifts(ptable, prows, tnight, internal_id, queue, reservation,
+              dry_run=0, strictly_successful=False,
+              check_for_outputs=True, resubmit_partial_complete=True,
+              z_submit_types=None, system_name=None):
+    """
+    Given a set of prows, this generates a processing table row, creates a batch script, and submits the appropriate
+    tilenight job given by descriptor. The returned ptable has all of these rows added to the
+    table given as input.
+
+    Args:
+        ptable, Table. The processing table where each row is a processed job.
+        prows list or array of dicts. Unsubmitted prestdstar jobs that are first steps in tilenight.
+        tnight, Table.Row. The row of the tilenight job on which the redshifts depend.
```

Comment:
Can you specify if it's a processing table row or an exposure table row? I believe it's a processing row (prow),

Reply:
Done.